jax/config.py

```diff
@@ -67,7 +67,7 @@ def add_option(self, name, default, opt_type, meta_args, meta_kwargs):
 
   def check_exists(self, name):
     if name not in self.values:
-      raise Exception("Unrecognized config option: {}".format(name))
+      raise AttributeError("Unrecognized config option: {}".format(name))
```

Comment:
This change was necessary because the pytest docstring finding machinary was erroring, similar to this issue: https://github.com/pytest-dev/pytest/issues/5080.